### PR TITLE
Added homework_13

### PR DESCRIPTION
**Pull request by olkhovyi**

### CODE REVIEW

File: Lesson_13/homework_13_json.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    "https://github.com/dntpanix/automation_qa/blob/0c61017dfea127e5e4a048bfc411f07e1c324b27/ideas_for_test/work_with_json/swagger.json"
+]
+
+# A function to check the validity of JSON files from URL
+def validate_json_from_urls(json_urls):
+    for url in json_urls:
+        try:
+            response = requests.get(url)
+            response.raise_for_status()  # We check whether the file was successfully downloaded
+            json_data = response.json()  # Trying to convert to JSON
```
Comment (edanilovets) [inline Lesson_13/homework_13_json.py:23]:
```suggestion
            response.json()  # Trying to convert to JSON
```
json_data variable is not used, so it is not needed here

File: Lesson_13/homework_13_csv.py [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+# Function to read a CSV file with the correct extension
+def read_csv(file_name, delimiter):
+    with open(file_name, mode='r', encoding='utf-8') as csvfile:
+        reader = csv.reader(csvfile, delimiter=delimiter)
+        data = [row for row in reader]
+    return data
+
+# Reading data from two CSV files
+data1 = read_csv('rmc.csv', delimiter=';')
+data2 = read_csv('r-m-c.csv', delimiter=';')
```
Comment (edanilovets) [inline Lesson_13/homework_13_csv.py:14]: this file has another delimiter

File: Lesson_13/homework_13_json.py (hunk 1 above)
Comment (olkhovyi) [reply Lesson_13/homework_13_json.py:23]: So I printed the result for verification. Deleted

File: Lesson_13/homework_13_csv.py (hunk 2 above)
Comment (olkhovyi) [reply Lesson_13/homework_13_csv.py:14]: The fact is that in one file the delimiter is "," and in the other ";". It doesn't matter which one I choose, the code works, but if there is no delimiter parameter, it gives an error about adding it.

File: Lesson_13/homework_13_csv.py [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+# Function to read a CSV file with the correct extension
+def read_csv(file_name, delimiter):
+    with open(file_name, mode='r', encoding='utf-8') as csvfile:
+        reader = csv.reader(csvfile, delimiter=delimiter)
+        data = [row for row in reader]
+    return data
+
+# Reading data from two CSV files
+data1 = read_csv('rmc.csv', delimiter=',')
```
Comment (edanilovets) [inline Lesson_13/homework_13_csv.py:13]: rmc.csv file has delimeter ,
r-m-c.csv has ;

Comment (olkhovyi) [reply Lesson_13/homework_13_csv.py:13]: That's right. I didn't fully understand why, but if you change the delimer to those in each file, the result file is duplicated twice. From line 132, the headings are duplicated again and the duplicate is displayed. Ok. We act on the task to remove duplicates. Corrected.

File: Lesson_13/homework_13_json.py [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+    "https://github.com/dntpanix/automation_qa/blob/0c61017dfea127e5e4a048bfc411f07e1c324b27/ideas_for_test/work_with_json/login.json",
+    "https://github.com/dntpanix/automation_qa/blob/0c61017dfea127e5e4a048bfc411f07e1c324b27/ideas_for_test/work_with_json/swagger.json"
+]
+
+# A function to check the validity of JSON files from URL
+def validate_json_from_urls(json_urls):
+    for url in json_urls:
+        try:
+            response = requests.get(url)
+            response.raise_for_status()  # We check whether the file was successfully downloaded
```
Comment (edanilovets) [inline Lesson_13/homework_13_json.py:22]: this code is not checking json, it only get data by url

**Pull request merged**